library/src/main/java/com/securepreferences/SecurePreferences.java

```diff
@@ -192,21 +194,23 @@ public void destroyKeys(){
     /**
      * Uses device and application values to generate the pref key for the encryption key
      * @param context
-     * @return String to be used as the AESkey Pref key
+     * @param iterationCount
+	 * @return String to be used as the AESkey Pref key
      * @throws GeneralSecurityException if something goes wrong in generation
      */
-	private static String generateAesKeyName(Context context) throws GeneralSecurityException
+	public String generateAesKeyName(Context context, int iterationCount) throws GeneralSecurityException

```

Comment:
@lala7573  Because I add parameter iteration count in `generateAesKeyName`, don't have to make public, as you said earlier. But, I think that logic to generate the pref key for the encryption key can editable for some case or someone's intention. Also I know `private static` is used for internal utilization not depended on business logic. So `public` is not necessary, but must be `protected` or `private` at least.